client/src/components/Dashboard.js

```diff
@@ -30,10 +31,19 @@ class Dashboard extends Component {
         },
         providerData: this.props.providerData,
         newEntry: false,
+        habitData: [],
+        user: '',
     };
 
-    componentDidMount() {
+    componentDidMount = () => {
         this.updateProviders(this.state.providerData);
+        const user = this.state.providerData
+        this.setState({ user: user[0].email })
+        axios.get('/api/habits/habit/5cb0191dcca50e00d2e290be')
```

Comment:
I'm not familiar with axios.... what's happening on this line?

It that hardcoded for testing purposes?

Reply:
That is the PR I pushed through for the GET request. I listed everything it's doing in that PR